src/matchers/lang/cfamily/java/JavaBody.ts

```diff
@@ -0,0 +1,20 @@
+import { Concat } from "../../../Concat";
```

Comment:
why lang/cfamily/java?

I would expect (as a user, my guess would be) lang/java, and then internally it could reference cfamily things